<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>930</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>930</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Chitragupt</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="CommandPlainTextEdit" name="command_line">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>911</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="header_title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;</string>
    </property>
    <property name="text">
     <string>C H I T R A G U P T</string>
    </property>
   </widget>
   <widget class="QLabel" name="header_quote">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>751</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>To keep talking for hours and still say nothing, is the skill of a master politician.</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>49</y>
      <width>911</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QScrollArea" name="content_scroll_area">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>621</width>
        <height>471</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>619</width>
         <height>469</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QScrollArea" name="todo_scroll_area">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>0</y>
        <width>281</width>
        <height>471</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>279</width>
         <height>469</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="cal_widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>621</width>
        <height>471</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid grey;</string>
      </property>
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>581</width>
         <height>431</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none; color:white;</string>
       </property>
      </widget>
     </widget>
     <widget class="QScrollArea" name="todo_scroll_area_2">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>0</y>
        <width>281</width>
        <height>471</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>279</width>
         <height>469</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid grey;</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <zorder>stackedWidget</zorder>
   <zorder>command_line</zorder>
   <zorder>header_title</zorder>
   <zorder>header_quote</zorder>
   <zorder>line</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommandPlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>commandplaintextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
